server/common/utils/helpers.js

```diff
@@ -326,6 +326,18 @@ const responseWithCohortDetails = (doc, userId) => {
   };
 };
 
+const runAsyncTasks = async (...tasks) => {
+  tasks.forEach(task => task());
+
+  try {
+    await Promise.all(tasks);
+  } catch (err) {
+    // eslint-disable-next-line no-console
+    console.error('Migration failed...');
```

Comment:
It would be nice to log the error as well to ease the debugging with `console.error(error)`.